### DISCUSSION TITLE
EACCES: permission denied, open '/var/wiki/data/cache ...

### DISCUSSION THREAD

**Discussion by WitheredHope** [requarks/wiki#2931] [Help / Questions] [answered]
If I start wiki.js with `node server` then I don't have any issues, but if I start it with `systemctl start wiki` the server starts fine and loads the homepage but if I try and view any other page I get:
 `EACCES: permission denied, open '/var/wiki/data/cache/something.bin'`

my /etc/systemd/system/wiki.service file is:

```[Unit]
Description=Wiki.js
After=network.target

[Service]
Type=simple
ExecStart=/usr/bin/node server
Restart=always
# Consider creating a dedicated user for Wiki.js here:
User=nobody
Environment=NODE_ENV=production
WorkingDirectory=/var/wiki

[Install]
WantedBy=multi-user.target
```

Info:
OS - Linux - Ubuntu (bionic) 18.04 x64
WikiJS Version - 2.5.170
Node.js - 12.16.2
PostgreSQL - 10.15

**Comment by scienceasdf** [comment, marked as answer]
Do you have read and wirte permission on /var/wiki folder? You can execute `'ls -l` command to see your permissions. To fix that, you could simply execute `sudo chown -R nobody /var/wiki`
